.gitlab-ci.yml

```diff
@@ -147,11 +174,12 @@ deploy:sls:master:
     <<: *tmpl_master_deploy_variables
   environment:
     name: production
-    url: https://$APP_URL
-  needs:
+    url: https://$BASE_URL
+  dependencies:
```

Comment:
Can we leave this as `needs`? Otherwise it'll wait for `test` before even starting the deploy, and the deploy is already quite slow :|

Reply:
Updated in the other job as well